plugins/services/src/js/stores/SDKEndpointStore.js

```diff
@@ -0,0 +1,154 @@
+import PluginSDK from "PluginSDK";
+
+import { SERVER_ACTION } from "#SRC/js/constants/ActionTypes";
+import AppDispatcher from "#SRC/js/events/AppDispatcher";
+import GetSetBaseStore from "#SRC/js/stores/GetSetBaseStore";
+import Util from "#SRC/js/utils/Util";
+
+import {
+  REQUEST_SDK_ENDPOINTS_SUCCESS,
+  REQUEST_SDK_ENDPOINTS_ERROR,
+  REQUEST_SDK_ENDPOINT_SUCCESS,
+  REQUEST_SDK_ENDPOINT_ERROR
+} from "../constants/ActionTypes";
+
+import SDKEndpointActions from "../events/SDKEndpointActions";
+import ServiceEndpoint from "../structs/ServiceEndpoint";
+
+class SDKEndpointStore extends GetSetBaseStore {
+  constructor() {
+    super(...arguments);
+
+    const self = this;
```

Comment:
Hey @bstavroulakis instead of doing this, let's change L#34 to use an arrow function
`this.dispatcherIndex = AppDispatcher.register((payload) => {`
and then we keep the parent context inside the arrow function and no longer need `self`.